README.md

```diff
@@ -74,18 +74,17 @@ You can find complete examples at [scanapi-examples][scanapi-examples]!
 
 ```bash
 $ scanapi --help
-Usage: scanapi [OPTIONS]
+Usage: scanapi [OPTIONS] [SPEC_PATH]
```

Comment:
Having `SPEC_PATH` before `OPTIONS` reinforces the idea that the former is required and the latter is optional, I think.

Reply:
Good idea. However, I looked for some documentation for this pattern and I didn't find it. Besides, I checked two terminal commands and the usage are in the same format we currently have:
- `gcc [OPTIONS] COMMAND`
- `docker [OPTIONS] COMMAND`
So IMHO there's no problem here to keep using this format :)

Reply:
Good examples. Although the idea of COMMAND (which is usually an imperative verb like: run, build, open, etc) is a bit different from SPEC_PATH (which sounds more like just another argument), I’m OK with both options.

Reply:
Good catch. I can see more clearly now. Thanks!

Reply:
Great discussion, I did not think about these points, thank you both! I agree that would be clearer to have [SPEC_PATH] before [OPTIONS], but I don't know how to do it actually 😂  This order ir generated by Click. I would need more investigation to figure out how to change it.